aspnetcore/client-side/bundling-and-minification/samples/BuildBundlerMinifierApp/BuildBundlerMinifierApp.csproj

```diff
@@ -4,7 +4,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="BuildBundlerMinifier" Version="2.6.362" />
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.1.4" />
+    <PackageReference Include="Microsoft.AspNetCore.All" />
```

Comment:
In most of these cases, we should convert to using the Microsoft.AspnetCore.App metapackage instead of .All. This is the default in our 2.1 templates.

Reply:
Yes, these are all un-converted 2.0 sample apps. They're tracked on #5495.